packages/cli/src/commands/client-task-queue/list.ts

```diff
@@ -0,0 +1,45 @@
+import { ClientTaskQueueService, State } from '@meeco/sdk';
+import { AuthConfig } from '../../configs/auth-config';
+import { authFlags } from '../../flags/auth-flags';
+import MeecoCommand from '../../util/meeco-command';
+
+export default class ClientTaskQueueList extends MeecoCommand {
+  static description = 'Read the client task that client is supposed to perform';
+  static examples = [`meeco client-task-queue:list -a path/to/auth.yaml`];
+
+  static flags = {
+    ...MeecoCommand.flags,
+    ...authFlags
+  };
+
+  static args = [{ name: 'state', required: false }];
```

Comment:
Is this an enum? If so might be worth providing an `options: []` and `description` property for it

Reply:
Perhaps by using keys of `State` from the sdk

Reply:
added two command options supressChangingState & state -s. you can verify by running meeco client-task-queue:list --help